lib/add-group-tangle.js

```diff
@@ -15,15 +15,10 @@ module.exports = function AddGroupTangle(server) {
   return function addGroupTangle(content, cb) {
     if (!content.recps) return cb(null, content)
 
-    if (!isCloakedMsg(content.recps[0])) return cb(null, content)
+    if (!isIdentityGroupSSBURI(content.recps[0])) return cb(null, content)
 
     getGroupTangle(content.recps[0], (err, tangle) => {
-      // NOTE there are two ways an err can occur in getGroupTangle
-      // 1. recps is not a groupId
-      // 2. unknown groupId,
-
-      // Rather than cb(err) here we we pass it on to boxers to see if an err is needed
-      if (err) return cb(null, content)
+      if (err) return cb(err)
```

Comment:
Took the liberty to actually error here. When validating against the spec we don't allow for tangle-less messages anyway so I think this comment probably isn't relevant here anymore.

Reply:
Good call, also the comment spoke about "boxers" and we don't use that anymore, it's a db1 thing

Reply:
uhhh.. I think we use boxers still, we just register them in plugins like ssb-box and ssb-box2 right?
Anyway, we just need to be clear about who catches encryption errors

@staltz (or anyone else) - do you agree that it's if there are 7 recps and one of them is bung, the encryption should fail (NOT the alternative which would be that it succeeds but only encrypts to 6, and you just don't know it wasn't encrypted to the 7th)